src/test/java/uk/gov/laa/ccms/caab/util/SoaModelUtils.java

```diff
@@ -555,7 +554,7 @@ public static ClientDetail buildClientDetail() {
                 .vexatiousLitigant(Boolean.FALSE)
                 .vulnerableClient(Boolean.FALSE))
             .specialConsiderations("special"))
-        .recordHistory(new ClientDetailRecordHistory());
+        .recordHistory(new uk.gov.laa.ccms.soa.gateway.model.RecordHistory());
```

Comment:
This class is imported, so don't need to fully qualify

Reply:
done